vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -21,9 +21,9 @@ if (-not ($validSKU -contains $windowsSKU)) {
 # You must install the April 22, 2021 servicing stack update (SSU) (KB5001407) before installing the latest cumulative update (LCU).
 # SSUs improve the reliability of the update process to mitigate potential issues while installing the LCU. 
 $global:patchUrls = @(
-    "http://download.windowsupdate.com/d/msdownload/update/software/updt/2021/05/windows10.0-kb5003217-x64_92c969426f41f135f842dd232a64e6a461a848ca.msu"
```

Comment:
_You must install the May 11, 2021 servicing stack update (SSU) (KB5003243) or the latest SSU (KB5003711) before installing the latest cumulative update (LCU)._

KB5003243 is installed in the vhd from the pipeline

Installed QFEs
	KB4601060 : Update          : http://support.microsoft.com/?kbid=4601060
	KB5003243 : Security Update : http://support.microsoft.com/?kbid=5003243
	KB5003646 : Security Update : http://support.microsoft.com/?kbid=5003646